qt_binder/tests/test_binder.py

```diff
@@ -59,7 +59,11 @@ def test_object_traits(self):
                   for k, v in self.Object.class_traits().items()}
         self.assertIsInstance(traits['deleteLater'], QtSlot)
         self.assertIsInstance(traits['destroyed'], QtSignal)
-        self.assertIsInstance(traits['destroyed_QObject'], QtSignal)
+        try:
+            self.assertIsInstance(traits['destroyed_QObject'], QtSignal)
+        except KeyError:
+            # on Python 3, key name is different
+            self.assertIsInstance(traits["destroyed_b'QObject*'"], QtSignal)
```

Comment:
The type of `t` is QtCore.QByteArray which does not seem to have a .decode() method.

I can do bytes(t).decode(), though and it seems to work.